### ISSUE TITLE
When initializing :chrome browser with :url specified, the default download directory is not set properly

### ISSUE THREAD

**Issue by RusHibaM** [watir/watir#812]
## Meta -
Watir Version: 6.14.0
Selenium Version: 3.14
Browser Version: Chrome 69.0.3497.100 (Official Build) (64-bit) 
Browser Driver Version:  2.42.591071 (0b695ff80972cc1a65a5cd643186d2ae582cd4ac)
OS Version: Linux 9b539a9f86cc 3.13.0-158-generic #208-Ubuntu SMP Fri Aug 24 17:07:38 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux (The one used by SeleniumHQ/docker-selenium container, we are actually using selenium/node-chrome-debug:3.14.0-helium and selenium/hub:3.14.0-helium)

## Expected Behavior -
When initializing the Watir::Browser with :url specified and browser type set to :chrome, it should set the default download directory correctly.

## Actual Behavior -
The default download directory is not set correctly

## Steps to reproduce -
```ruby
prefs = {
    download: 
        {
            prompt_for_download: false,
            default_directory: download_directory
        } 
}
b = Watir::Browser.new :chrome, url: "http://0.0.0.0:32768/wd/hub",  options: {prefs: prefs}l
```
We are using the selenium docker and the 32768 is the port that binds to the 4444 port of the hub container. 
After the browser is initialized, use the VNC window to check the browser. In the advanced setting, the download directory is still /home/seluser/Downloads, not the download_directory given.

Did a few debugging and I found that when :url is given, the browser type is always overrided to :remote (https://github.com/watir/watir/blob/master/lib/watir/capabilities.rb#L15).

This seems to lead to the Selenium::Chrome::Serivce never being used, as the Selenium::WebDriver::Remote::Driver is actually used, instead of Selenium::WebDriver::Chrome::Driver.

I 'hacked' the code a little by change the line 15 of the (https://github.com/watir/watir/blob/master/lib/watir/capabilities.rb#L15) to:
```ruby
@selenium_browser = browser == :remote ? :remote : browser
```

After this, re-run the code I provide. And this time, the default download directory could be set successfully.

This seems to be a problem for people using the selenium docker, as I will need to specify the url when initialize the browser. Not sure if this is the expected behavior.

Is there a bug that makes :remote browser not able to set the default directory? Or does this mean that I have to initialize my own Selenium::WebDriver and pass it into the constructor of the Watir::Browser?

**Comment by RusHibaM** [comment]
The tweak I mentioned may not work correctly if the containers are truly 'remote' (on some other machines). 

So what I am thinking about is:

1. Should :remote browser also sets the download directory correctly? Is it really necessary to make a :remote browser when :url is given? It could happen that a local port other than 4444 is used

2. It will be nice to have documentation talking about how to initialize a :remote chrome browser with customized default download directory

**Comment by RusHibaM** [comment]
FYI: Did a few other experiments about the :remote browser. It shows that forcing to use the :chrome type of browser when trying to talk to a hub container that is hosted on another machine seems to be working properly. With the tweak, our web GUI automation is now running smoothly with chrome.

**Comment by titusfortner** [comment]
It's a little weird. ":remote" is used with Selenium any time it needs to go through a selenium server instead of accessing the driver directly.

Every time `:remote` is used, there must be a `:url`, and vice versa. The server will start the driver based on the capabilities it receives from the Options/Capabilities class you send it. My idea with the capabilities class was to avoid the duplication, and your code is exactly what I intended for Watir to support. Except there is indeed a bug, where the options aren't properly getting built when it is remote.

I'm fixing it in Watir 6.19.

You can build what you need from Capabilities class, but that would be a pain. For now, this works:
So for instance, Selenium wants to see 
```ruby
options = Selenium::WebDriver::Chrome::Options.new(prefs: {downloads: download_prefs})
browser = Watir::Browser.new :remote, url: server_url, options: options)
```

**Comment by titusfortner** [comment]
This has been fixed in 6.19, your example should now work. Further, Chrome is default if nothing is specified, so you can even do:
```ruby
b = Watir::Browser.new url: "http://0.0.0.0:32768/wd/hub",  options: {prefs: prefs}
```
